keps/sig-node/2625-cpumanager-policies-thread-placement/README.md

```diff
@@ -300,6 +267,19 @@ The graduation Criteria of options is described below:
 - [X] Allowing time for feedback (1 year) on the policy option.
 - [X] Risks have been addressed.
 
+### Removal of the CPUManagerPolicyAlphaOptions and CPUManagerPolicyBetaOptions feature gates
+
+We plan to remove the `CPUManagerPolicyAlphaOptions` and `CPUManagerPolicyBetaOptions` after all options graduated to stable, after a feature cycle passes without new planned options, and not before 1.28, to give ample time to the work in progress option to graduate at least to beta.
```

Comment:
1.28 has passed and I believe we've added more options, should this section be updated? not necesssarily a GA blocker for the KEP but could be useful to make clear our intention with these fields

Reply:
Yes, makes sense. Rephrased and rearranged with the other content of this section